teachopencadd/talktorials/T014_binding_site_detection/talktorial.ipynb

```diff
@@ -0,0 +1,2116 @@
+{
```

Comment:
<p>The <code>biopandas</code>  API should allow us now  to replace the private method call <code>_construct_df</code>, see https://github.com/rasbt/biopandas/pull/72. I will take care of this after the review here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>Thanks, again!</p>

Reply:
Cannot update this yet because the changes from that PR are not yet part of a new release. Leaving a TODO comment in the code.